src/diffpy/utils/tools.py

```diff
@@ -206,3 +209,47 @@ def get_package_info(package_names, metadata=None):
         pkg_info.update({package: importlib.metadata.version(package)})
     metadata["package_info"] = pkg_info
     return metadata
+
+
+def compute_mu_using_xraydb(sample_composition, energy, sample_mass_density=None, packing_fraction=None):
```

Comment:
change name from `density` to `sample_mass_density`, set default of `packing_fraction` to None